docs/configuration-reference/components/aws-ebs-csi-driver.md

```diff
@@ -0,0 +1,47 @@
+# CSI Driver for Amazon EBS configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Applying](#applying)
+* [Destroying](#destroying)
+
+## Introduction
+
+The [CSI Driver for Amazon EBS](https://github.com/kubernetes-sigs/aws-ebs-csi-driver)
```

Comment:
I think we could make this description a bit more user friendly, to describe, that this component provides storage class for AWS, baked by Amazon EBS volumes.

We should also mention, that when cluster is destroyed, volumes must be cleaned up manually.